data/p/selected-slack-and-linear-integrations.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "2 selected Slack and Linear integrations"
+date: "2022-10-20"
+tags: ["developer-productivity", "slack", "linear"]
+draft: false
+summary: In this blog post, we will compare the top Linear Slack integrations and provide a tutorial on how to install the official Linear Slack integration. We will also discuss the different alternatives you have out there and how they can help improve productivity in your business.
+layout: PostLayout
+canonicalUrl: slack-linear-integration
+image: "/blog/static/images/linear/linear-slack-integrations.jpg"
+---
+
+In this blog post, we will compare the top Linear Slack integrations and provide a tutorial on how to install the official Linear Slack integration. We will also discuss the different alternatives you have out there and how they can help improve productivity in your business.
+
+## 1. Official Linear Slack integration
+
+### How does the official Linear Slack integration work?
+
+The Linear Slack integration is one of the most popular Slack integrations. It allows you to track and manage your Linear team's progress directly from Slack. With the Linear Slack integration, you can:
+
+- Create and view Linear items
+
+- Assign Linear items to team members
+
+- Set due dates and reminders for Linear items
+
+- View item details and comments
+
+- Get notifications when Linear items are created or updated
+  <ImageContainer
+    alt="linear-official-slack-integration"
+    src="/blog/static/images/linear/linear-official-slack-integration.jpeg"
+    classNameDiv=""
+    classNameImage=""
+    width={679}
+    height={424}
+  />
+
+### How to set up notifications?
+
+Slack notifications are set up in many places around the app. Anyone can configure them as long as the Slack integration is enabled. To set up:
+
+team notifications on either the team's general settings or the integration settings page. Authenticate with Slack and then select which channel to post them in.
+project notifications from the pages of specific projects. Click the bell in the title bar on the top right of the app, sign in to Slack, and then select which channel to post them to.
+
+You can configure personal Slack notifications in the notification settings. Authenticate with Slack and then select which notifications you want to receive.
+
+Once you have installed the Linear Slack integration, you will be able to access all of its features from within your Slack workspace. To get started, simply type '/linear' into the Slack search bar and select the 'Create Item' option. From there, you can create new items, assign them to team members, set due dates, and add comments - all without ever leaving Slack!
+
+If you need any assistance along the way, simply type '/linear help' into Slack and you will be prompted with all the linear possibilities.
+
+## 2. Zapier to integrate Slack and Linear
+
+Zapier has a Linear and Slack integrations. This can add additional granularity to what you would get with the official integration.
```

Comment:
```suggestion
Zapier has a Linear and Slack integration. This can add additional granularity to what you would get with the official integration.
```